src/main/kotlin/org/stellar/walletsdk/util/ValidateTransaction.kt

```diff
@@ -0,0 +1,18 @@
+package org.stellar.walletsdk.util
+
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+fun validateTransaction(transaction: Transaction, server: Server) {
```

Comment:
Can we rename this `validateSufficientBalance()` or something similar? The name is too generic for what is does today.